### ISSUE TITLE
Double splat operator error

### ISSUE THREAD

**Issue by JonathanOfTheWell** [rubocop/rubocop#9866]
## Expected behavior
Running bundle exec on this file to not throw an error
``` ruby
# frozen_string_literal: true
def foo(first_param, **multipleparams, &block)
end

def bar
  first_param = "param"
  multipleparams = {
    param1: "value1",
    param2: "value2",
  }

  foo(first_param, **multipleparams) do
    return
  end
end
```

## Actual behavior

```
An error occurred while Layout/HashAlignment cop was inspecting [file].
undefined method `loc' for nil:NilClass
[file_path]/.gem/ruby/2.7.1/gems/rubocop-1.16.1/lib/rubocop/cop/layout/hash_alignment.rb:221:in `autocorrect_incompatible_with_other_cops?'
[file_path]/.gem/ruby/2.7.1/gems/rubocop-1.16.1/lib/rubocop/cop/layout/hash_alignment.rb:206:in `on_hash'
[file_path]/.gem/ruby/2.7.1/gems/rubocop-1.16.1/lib/rubocop/cop/commissioner.rb:100:in `public_send'
[file_path]/.gem/ruby/2.7.1/gems/rubocop-1.16.1/lib/rubocop/cop/commissioner.rb:100:in `block (2 levels) in trigger_responding_cops'
[file_path]/.gem/ruby/2.7.1/gems/rubocop-1.16.1/lib/rubocop/cop/commissioner.rb:160:in `with_cop_error_handling'
[file_path]/.gem/ruby/2.7.1/gems/rubocop-1.16.1/lib/rubocop/cop/commissioner.rb:99:in `block in trigger_responding_cops'
[file_path]/.gem/ruby/2.7.1/gems/rubocop-1.16.1/lib/rubocop/cop/commissioner.rb:98:in `each'
[file_path]/.gem/ruby/2.7.1/gems/rubocop-1.16.1/lib/rubocop/cop/commissioner.rb:98:in `trigger_responding_cops'
```
## Steps to reproduce the problem

execute the following command with `[file]` replaced by the path to a file containing the above code snippet
``` bash
bundle exec rubocop [file]
```

## RuboCop version

1.16.1 (using Parser 3.0.1.1, rubocop-ast 1.7.0, running on ruby 2.7.1 x86_64-darwin20)

**Comment by dvandersluis** [comment]
Hi @JonathanOfTheWell thanks for the report! This should be fixed already by #9862.
